cfme/cloud/keypairs.py

```diff
@@ -0,0 +1,34 @@
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.web_ui import Quadicon, Region, flash
+from utils.pretty import Pretty
+from functools import partial
+from cfme.web_ui import toolbar as tb
+from utils.wait import wait_for
+
+
+details_page = Region(infoblock_type='detail')
```

Comment:
I would say do not use this Region, but make a `load_details` method on the KeyPairs with the same synopsis like https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/provider.py#L432 and then include a SummaryMixin from `cfme.common`